gagofure/posttimes
# Add Navbar

#### What does this PR do?
- This PR adds a Navbar to the App 

#### Description of tasks to be completed?
- Install react-router-dom 
- add component folder and home, about and contact pages
- add Navbar component and route the pages 
- add doc folder with commit and PR files

#### How should this be manually tested?
  - Clone the repo with `clone https://github.com/gagofure/posttimes.git`
  - Checkout to branch `feat-navbar`
  - Install dependencies `yarn  or npm install`
  - Run application `yarn or npm start`
  - you should see a navbar on the browser

#### Any background context you want to provide?
- You would need npm or yarn to run this application

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
 N/A
